x-pack/plugins/security_solution/public/management/pages/index.tsx

```diff
@@ -76,43 +75,53 @@ const ResponseActionsTelemetry = () => (
 );
 
 export const ManagementContainer = memo(() => {
-  const { loading, canAccessEndpointManagement, canReadActionsLogManagement } =
-    useUserPrivileges().endpointPrivileges;
+  const {
+    loading,
+    canAccessEndpointManagement,
+    canReadPolicyManagement,
+    canReadBlocklist,
+    canReadTrustedApplications,
+    canReadEventFilters,
+    canReadActionsLogManagement,
+  } = useUserPrivileges().endpointPrivileges;
 
   // Lets wait until we can verify permissions
   if (loading) {
     return <EuiLoadingSpinner />;
   }
 
-  if (!canAccessEndpointManagement) {
-    return (
-      <>
-        <Route path="*" component={NoPermissions} />
-        <SpyRoute pageName={SecurityPageName.administration} />
-      </>
-    );
-  }
-
   return (
     <Switch>
-      <Route path={MANAGEMENT_ROUTING_ENDPOINTS_PATH} component={EndpointTelemetry} />
-      <Route path={MANAGEMENT_ROUTING_POLICIES_PATH} component={PolicyTelemetry} />
-      <Route path={MANAGEMENT_ROUTING_TRUSTED_APPS_PATH} component={TrustedAppTelemetry} />
-      <Route path={MANAGEMENT_ROUTING_EVENT_FILTERS_PATH} component={EventFilterTelemetry} />
+      {canAccessEndpointManagement && (
```

Comment:
same here - can we stop using `canAccessEndpointManagment`?

Reply:
Should we use `canReadEndpointList` instead?